packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -245,7 +245,20 @@ const makeRenderer = (
 					initialNow: new Date().getTime(),
 					isProdApi: server.settings.app.api.isProd,
 					isQL: parseMemberCookie(state).ql === 'true',
-					memberId: parseMemberCookie(state).id,
+					/**
+					 * Original tickets that caused need:
+					 * 
+					 * @see https://meetup.atlassian.net/browse/MG-2473
+					 * @see https://meetup.atlassian.net/browse/MG-2441
+					 * 
+					 * For segmentation and rollout purposes, it is extremely advantageous
+					 * to have the member object available pre query to do split tests
+					 * based off of things like the member's id or location.
+					 * 
+					 * If the member is not logged in, this will return an empty object
```

Comment:
no need to document what `parseMemberCookie` does here. Also, just nitpick, `{ id: 0 }` is not an 'empty object'.